### ISSUE TITLE
How change the db.sqlite to download the files in 'D' drive, instead of the previous 'E' drive? 

### ISSUE THREAD

**Issue by esotericer1** [Nandaka/PixivUtil2#1192]
Same issue with the previous issue #1164 : 
After changing the PC and the main Drive has been changed from 'E' to 'D', downloader seems to be having problems, due to db.sqlite has been 'conditioned' to download at E drive. 
whenever I try downloading from list.txt, it just creates 'Error medium page' chrome files, with messages like

"can't find alloted path 'D:\\' " 

my usual list.txt has been something like this :
13286336 .\02
18145190 .\03
10567775 .\01

and it would download all new files in folders '01', '02' and '03', under the subfolders with the each artist's name.

but when I tried it with the new PC it doesn't download any images, or download entire images in one single folder, not putting them neatly in folders for artist names inside 01, 02, 03, etc.
I had pixivdownloader2 for a long time and forgot how I managed to set download paths. so please excuse me if my question is the most basic of stuffs.

I tried changing the config file and chaninging 'downloadListDirectory' or 'rootDirectory' to 'D:\Pixiv2', but it doesn't seem to work. 

TL;DR, any ideas on downloading new files that are not in my db.sqlite into neat folders, the only difference being the drive is 'D' now, instead of 'E' ?
and without having to create an entirely new db.sqlite.

Also please forgive me if my explanations were hard to understand. English is not my first language ;;

version : latest v20221029

**Comment by Nandaka** [comment]
weird, it should work. Do you extract the zip file first and run it from there (try to click the exe directly, not using shortcut)

![image](https://user-images.githubusercontent.com/836867/201936235-7b3504b6-c8c2-458b-a238-01970932a6cb.png)

can you upload the log file? Fyi, you can change the drive letter using Computer Management
![image](https://user-images.githubusercontent.com/836867/201936644-ffb283c9-e3af-4af9-ba30-24cfde464718.png)

**Comment by esotericer1** [comment]
Changing the drive name seems to work. Thank you!

2022년 11월 15일 (화) 오후 10:53, Nandaka ***@***.***>님이 작성:
